src/components/row.js

```diff
@@ -334,4 +345,20 @@ function Row(props) {
   );
 }
 
+const tooltipStyle = {
```

Comment:
@ameya-pandilwar can you move the style to CSS file? The pattern you followed by storing it to a var is not used anywhere else in the file.

Reply:
@tannakartikey sure, will change it to an inline style to make it consistent with the existing style